.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
 sudo: true
 install:
   - "curl -O https://raw.githubusercontent.com/Melissa-AI/Melissa-Core/master/setup/travis-build-core.sh && chmod +x travis-build-core.sh && ./travis-build-core.sh"
-  - "cp data/memory.db.default data/memory.db"
```

Comment:
this is what i propose to copy memory. the strategy is to use appdirs and have the program manage the database on use app folder

```python
# main file
from appdirs import user_data_dir

def main():
    # ...
    default_db = ...
    db_path = os.path.join(appdirs('melissa', 'tanay'), 'memory.db')
    if not os.path.isfile(db_path):
        shutil.copyfile(default_db, db_path)
    ...
```
for action module that need them can use either the global variable on main file or use appdirs to get db path.